deploy/scripts/src/btm/test.ts

```diff
@@ -1,29 +1,148 @@
+import { sleep } from 'k6'
 import { type Options } from 'k6/options'
 import { selectProfile, type ProfileList, describeProfile } from '../common/utils/config/load-profiles'
 import { uuidv4 } from '../common/utils/jslib/index'
 import { AWSConfig, SQSClient } from '../common/utils/jslib/aws-sqs'
 
+const loadProfileVar = {
+  smokeMaxVU: __ENV.DATA_BTM_PROFILE_SMOKE_MAX_VU,
+  smokeTarget: __ENV.DATA_BTM_PROFILE_SMOKE_TARGET,
+  smokeDuration: __ENV.DATA_BTM_PROFILE_SMOKE_DURATION,
+
+  loadMaxVU1: __ENV.DATA_BTM_PROFILE_LOAD_MAX_VU1,
+  loadTarget1: __ENV.DATA_BTM_PROFILE_LOAD_TARGET1,
+  loadDuration: __ENV.DATA_BTM_PROFILE_LOAD_DURATION,
+
+  loadMaxVU2: __ENV.DATA_BTM_PROFILE_LOAD_MAX_VU2,
+  loadTarget2: __ENV.DATA_BTM_PROFILE_LOAD_TARGET2,
+
+  loadMaxVU3: __ENV.DATA_BTM_PROFILE_LOAD_MAX_VU3,
+  loadTarget3: __ENV.DATA_BTM_PROFILE_LOAD_TARGET3,
+
+  loadMaxVU4: __ENV.DATA_BTM_PROFILE_LOAD_MAX_VU4,
+  loadTarget4: __ENV.DATA_BTM_PROFILE_LOAD_TARGET4,
+
+  loadMaxVU5: __ENV.DATA_BTM_PROFILE_LOAD_MAX_VU5,
+  loadTarget5: __ENV.DATA_BTM_PROFILE_LOAD_TARGET5,
+
+  loadMaxVU6: __ENV.DATA_BTM_PROFILE_LOAD_MAX_VU6,
+  loadTarget6: __ENV.DATA_BTM_PROFILE_LOAD_TARGET6,
+
+  loadMaxVU7: __ENV.DATA_BTM_PROFILE_LOAD_MAX_VU7,
+  loadTarget7: __ENV.DATA_BTM_PROFILE_LOAD_TARGET7
```

Comment:
The load profiles shouldn't be parameterised, we should be clear about changes to the load profiles we are defining, add another profile if one is needed. This will ensure that our tests are reproducible and we are clear on the profiles we are testing against.

Reply:
Done